.github/workflows/gh-pages.yml

```diff
@@ -1,3 +1,9 @@
+# SPDX-FileCopyrightText: Organisation européenne pour la recherche nucléaire (CERN) <https://www.cern.ch>
```

Comment:
```suggestion
# SPDX-FileCopyrightText: CERN
```
I think CERN is enough here. See also: https://indico.cern.ch/category/4251/attachments/101/505/OSL-2012-01-Open_Source_Licences_at_CERN-Short_version.pdf